app/[lang]/components/members.tsx

```diff
@@ -65,61 +23,7 @@ const Members = ({ dictionary }: Props) => {
           {dictionary.home.members.subtitle}
         </p>
 
-        <div className="grid grid-cols-1 gap-8 mt-8 xl:mt-10 md:grid-cols-2 xl:grid-cols-4">
-          {randomMembers.map((member, i) => (
-            <div
-              className="flex flex-col items-center p-8 transition-colors duration-300 transform border border-slate-400 rounded-xl hover:border-transparent group hover:bg-red-500"
-              key={i}
-            >
-              <div className="overflow-hidden rounded-full ring-4 ring-red-500">
-                <Image
-                  className="object-cover w-32 h-32  transition-transform duration-300 hover:scale-125"
-                  src={`https://github.com/${member.github}.png`}
-                  alt="Member image"
-                  height={128}
-                  width={128}
-                />
-              </div>
-
-              <h1 className="mt-4 text-2xl font-semibold text-gray-700 capitalize group-hover:text-white">
-                {member.name}
-              </h1>
-
-              <p className="mt-2 text-gray-500 capitalize group-hover:text-gray-300 text-center">
-                {member.role}{' '}
-              </p>
-              {member.company && (
-                <span className="font-bold text-red-500 group-hover:text-slate-200">
-                  {member.company}
-                </span>
-              )}
-
-              <div className="flex mt-3 -mx-2">
-                {Object.values(icons).map((icon, i) => {
-                  if (!icon.shouldRender(member)) return null;
-
-                  return (
-                    <a
-                      key={i}
-                      href={icon.getUrl(member)}
-                      className="mx-2 text-gray-600 hover:text-gray-500 group-hover:text-white transition duration-300 transform hover:-translate-y-1"
-                      target="_blank"
-                    >
-                      <svg
-                        xmlns="http://www.w3.org/2000/svg"
-                        viewBox={icon.viewBox}
-                        className="w-6 h-6 fill-current"
-                        fill="none"
-                      >
-                        {icon.svgContent}
-                      </svg>
-                    </a>
-                  );
-                })}
-              </div>
-            </div>
-          ))}
-        </div>
+        <MembersList classNames="grid grid-cols-1 gap-8 mt-8 xl:mt-10 md:grid-cols-2 xl:grid-cols-4"></MembersList>
```

Comment:
FIXED

Reply:
It was designed in a way that you can define different grid distributions depending on the context. This is because classNames can be passed to MembersList as prop. But there is a default grid class in MembersList, so passing it always is not needed. I see you added a `BlogMembersList` that is good 👍